docker-compose.yml

```diff
@@ -24,6 +24,8 @@ services:
       EUREKA_CLIENT_SERVICEURL_DEFAULTZONE: "http://discovery-server:8761/eureka/"
       SPRING_CLOUD_CONFIG_LABEL: "main"
       SPRING_CLOUD_CONFIG_URI: "http://config-server:8888"
+      SITE-SERVICE_RIBBON_LISTOFSERVERS: "http://site-service:8083"
```

Comment:
This is something completely new.
why we need it? is it a must?

Reply:
We need this only for local use. This disables the @LoadBalanced annotation and replaces it with a defined url (like we need in docker compose or local run e.g). I add a comment and no need to put this env in prod

Reply:
can it be removed? because you have it in the -local.yml file

Reply:
In the local.yml I put "localhost:8082" for example if someone would run it from the inteliji, and for docker compose we need the service name + port